src/components/Button.jsx

```diff
@@ -1,8 +1,15 @@
-const Button = () => {
+import Link from "next/link";
+
+const Button = ({ link, target }) => {
   return (
-    <div className="">
-      <p>Button</p>
-    </div>
+    <Link
+      className="h-20 w-13 text-4xl text-swim-black bg-swim-yellow px-10 py-5 font-bold text-xl text-center hover:opacity-50 active:opacity-40 z-10"
+      href="https://acm.cs.ucr.edu/"
```

Comment:
put the link prop inside of href, do the same for target

Reply:
I have taken care of the following comments, could you double check to see if all the issues have been resolved?